contrib/test_jenkins.sh

```diff
@@ -213,6 +214,16 @@ for target_v in "${target_list[@]}"; do
 	        rc=$((rc + $ret))
 	    fi
     fi
+    if [ 9 -lt "$jenkins_opt_exit" -o "$rc" -eq 0 ]; then
```

Comment:
I think that it is better to move this quick check before compilation and unit tests to speed-up error detection.